From: Davide Mancusi <arekfu@gmail.com>
Subject: Generate colour Makefiles if $TERM=screen-256color-bce
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=627853
Acked-By: Modestas Vainius <modax@debian.org>
Forwarded: no
Last-Update: 2011-05-24
Origin: other, http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=5;filename=cmake_screen-256color-bce.patch;att=1;bug=627853

Generated Makefiles do not use color highlighting if $TERM is set to
`screen-256color-bce'. The patch fixes this.

--- a/Source/kwsys/Terminal.c
+++ b/Source/kwsys/Terminal.c
@@ -163,6 +163,7 @@ static const char* kwsysTerminalVT100Nam
   "rxvt-unicode-256color",
   "screen",
   "screen-256color",
+  "screen-256color-bce",
   "screen-bce",
   "screen-w",
   "screen.linux",
